yarn.lock

```diff
@@ -9624,8 +9605,8 @@ mkdirp@^0.5.1, mkdirp@^0.5.3, mkdirp@^0.5.5, mkdirp@~0.5.1:
 
 mobile-drag-drop@^2.3.0-rc.2:
   version "2.3.0-rc.2"
-  resolved "https://registry.yarnpkg.com/mobile-drag-drop/-/mobile-drag-drop-2.3.0-rc.2.tgz#00d6e85e04512a620fd5357366e8786bd29aa7aa"
-  integrity sha512-4rHP0PUeWkSp0O3waNHPQZCHeZnLu8bE59MerWOnZJ249BCyICXL1WWp3xqkMKXEDFYuhfk3bS42bKB9IeN9uw==
+  resolved "https://artifacts.jpmchase.net/artifactory/api/npm/npm/mobile-drag-drop/-/mobile-drag-drop-2.3.0-rc.2.tgz#00d6e85e04512a620fd5357366e8786bd29aa7aa"
```

Comment:
@steveukx all these packages have been installed against the internal registry, while modular should use the regular yarn registry (`registry.yarnpkg.com`) - could you 
1) point your registry to the public one
2) clean your yarn cache and node_modules
3) restore the yarn.lock from main
4) re-run `yarn` to update it
5) commit

Reply:
Thank you for spotting that! This is done